pages/protect-your-data.jsx

```diff
@@ -24,73 +23,61 @@ var LearningObjectives = (
+        link: "https://mozilla.github.io/curriculum-final/privacy-basics/session01-ip-tracer.html"
       },
       {
-        title: "Understanding security",
+        title: "Bad Password Collage",
         image1x: "/img/pages/protect-your-data/img-password.png",
         image2x: "/img/pages/protect-your-data/img-password@2x.png",
-        subtitle: "Understanding web mechanics",
-        description: "Learners investigate the differences between weak and strong passwords, and create a collage to document bad password practices.",
-        link: "http://mozilla.github.io/webmaker-curriculum/ProtectingYourData/session01-password-collage.html"
-      }
-    ]
-  },
-  {
-    title: "Writing the Web",
-    activities: [
+        subtitle: "Evaluate, Protect",
+        description: "You will learn about weak and strong passwords and test the strength of different passwords using tools called password meters. You will also make a &#34;bad password collage,&#34; learning web literacy skills like <strong>evaluate</strong> and <strong>protect</strong>.",
+        link: "https://mozilla.github.io/curriculum-final/privacy-basics/session02-bad-password-collage.html"
+      },
       {
         title: "Cookies and Third-Party Tracking",
         image1x: "/img/pages/protect-your-data/img-cookies.png",
         image2x: "/img/pages/protect-your-data/img-cookies@2x.png",
-        subtitle: "Understanding privacy and open practices",
-        description: "Using a free and open source tool called Mozilla Lightbeam, learners see how cookies and third-party trackers monitor their everyday tasks on the web. Then they divide into teams and race to see who can gather the most trackers.",
-        link: "http://mozilla.github.io/webmaker-curriculum/ProtectingYourData/session02-lightbeam.html"
+        subtitle: "Navigate, Open Practice, Protect",
+        description: "You will use a privacy tool called Mozilla Lightbeam to see how websites track your movement online, learning web literacy skills like <strong>navigate</strong>, <strong>open practice</strong>, and <strong>protect</strong>.",
+        link: "https://mozilla.github.io/curriculum-final/privacy-basics/session03-cookies-and-third-party-tracking.html"
       },
       {
-        title: "Draw Secure Passwords",
-        image1x: "/img/pages/protect-your-data/img-password.png",
-        image2x: "/img/pages/protect-your-data/img-password@2x.png",
-        subtitle: "Understanding privacy and security",
-        description: "Learners generate pass-phrases, pronounceable and random passwords based on best practices. Then they create a survey to pass along their knowledge to friends and family.",
-        link: "http://mozilla.github.io/webmaker-curriculum/ProtectingYourData/session02-secure-passwords.html"
-      }
-    ]
-  },
-  {
-    title: "Participating on the Web",
-    activities: [
+        title: "Create Secure Passwords",
+        image1x: "/img/pages/protect-your-data/pass-phrase.jpg",
```

Comment:
I don't know where this image came from, but 
![](https://media.giphy.com/media/dOJt6XZlQw8qQ/giphy.gif)